LICENSE.txt

```diff
@@ -2119,6 +2119,11 @@ DEALINGS IN THE SOFTWARE.
 
 --------------------------------------------------------------------------------
 
+Some source code from Folly (https://github.com/facebook/folly) has been adapted
+for PyArrow. Folly is released under the Apache License, Version 2.0.
```

Comment:
Can you mention the file name, as for other notices? Also, not sure why "PyArrow" specifically.

Reply:
Done.